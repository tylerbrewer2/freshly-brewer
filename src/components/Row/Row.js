import React from 'react';

const Row = ({ className, children, middle}) => {
  const middleClass = middle ? 'middle-xs' : undefined;

  return(
    <div className={`${className} ${middleClass} row`}>
      {children}
    </div>
  )
}

export default Row;