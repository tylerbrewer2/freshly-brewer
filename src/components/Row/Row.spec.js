import React from 'react';
import renderer from 'react-test-renderer';

import Row from './Row';

it('renders correctly', () => {
  const tree = renderer.create(
    <Row middle>
      Test children
    </Row>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
