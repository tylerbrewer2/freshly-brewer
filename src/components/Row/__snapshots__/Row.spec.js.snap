exports[`test renders correctly 1`] = `
<div
  className="undefined middle-xs row">
  Test children
</div>
`;
